client/src/components/AuditAdmin/Setup/Review/StandardizeContestChoiceNamesDialog.tsx

```diff
@@ -0,0 +1,244 @@
+import {
+  Button,
+  Callout,
+  Classes,
+  Colors,
+  Dialog,
+  HTMLSelect,
+  HTMLTable,
+} from '@blueprintjs/core'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import FormButton from '../../../Atoms/Form/FormButton'
+import { IContest } from '../../../../types'
+import { IContestChoiceNameStandardizations } from '../../../useContestChoiceNameStandardizations'
+import { IJurisdiction } from '../../../useJurisdictions'
+
+export function isContestChoiceNameStandardizationComplete(
+  standardizations: IContestChoiceNameStandardizations
+): boolean {
+  for (const jurisdictionStandardizations of Object.values(standardizations)) {
+    for (const contestStandardizations of Object.values(
+      jurisdictionStandardizations
+    )) {
+      for (const standardizedChoiceName of Object.values(
+        contestStandardizations
+      )) {
+        if (standardizedChoiceName === null) {
+          return false
+        }
+      }
+    }
+  }
+  return true
+}
+
+const Table = styled(HTMLTable)`
+  background: ${Colors.WHITE};
+  border: 1px solid ${Colors.LIGHT_GRAY1};
+  width: 100%;
+
+  tr th,
+  tr td {
+    vertical-align: middle;
+    word-wrap: break-word;
+  }
+`
+
+interface IDialogProps {
+  contest: IContest
+  isOpen: boolean
+  jurisdictionsById: { [id: string]: IJurisdiction }
+  onClose: () => void
+  standardizations: IContestChoiceNameStandardizations
```

Comment:
I wonder if it would make things a bit simpler if this data was nested like so: `{ contestId: { jurisdictionId: choiceStandardizations }}` since the dialog pertains to just one contest. Then you could just pass in a single contest's sub-object. Might make the backend more complicated such that it's not worth the tradeoff, but thought I'd mention the idea.

Reply:
Good nudge! I actually tried this when I was first implementing this feature but then found that I had to convert back to  `{ jurisdictionId: { contestId: choiceStandardizations }}` on the backend as you've noted

I ultimately found that the jurisdiction-first schema required less conversion than the contest-first schema